spec/commands/down-spec.js

```diff
@@ -1,82 +1,70 @@
-import proxyquire from 'proxyquire';
-import sinon from 'sinon';
-import { expect } from 'chai';
+import DatabaseClient from '../../lib/database/database-client';
+import { readMigrationFiles } from '../../lib/migrations/migration-files';
+import down from "../../lib/commands/down";
 
-import createMockDatabaseClient from '../helpers/create-mock-database-client.js';
-import createMockLogger from '../helpers/create-mock-logger';
+jest.mock('../../lib/database/database-client');
+
+// Suppress the logging.
+jest.mock('../../lib/utilities/logger');
+
+// Mock migration files.
+jest.mock('../../lib/migrations/migration-files');
 
 describe("down", () => {
-  let down, MockDatabaseClient, readMigrationFilesStub, migrationsDirectory, databaseOptions,
-    migration;
+  let databaseOptions, migrationsDirectory, migration;
 
   beforeEach(() => {
-    readMigrationFilesStub = sinon.stub();
-    MockDatabaseClient = createMockDatabaseClient();
-
-    down = proxyquire('../../lib/commands/down', {
-      '../database/database-client': { default: MockDatabaseClient },
-      '../migrations/migration-files': { readMigrationFiles: readMigrationFilesStub },
-      '../utilities/logger': { default: createMockLogger() }
-    }).default;
+    jest.resetAllMocks();
 
     migration = {
       direction: 'down',
       basename: '17760704000000-hello.sql',
       timestamp: '17760704000000'
     };
 
-    readMigrationFilesStub.resolves([ migration ]);
-
     databaseOptions = {};
+
     migrationsDirectory = "/tmp/migrations";
+
+    readMigrationFiles.mockReturnValue([ migration ]);
   });
 
   it("creates a new client", async () => {
     await down(databaseOptions, migrationsDirectory);
-    expect(MockDatabaseClient).to.have.been.calledWithNew;
+    expect(DatabaseClient).toHaveBeenCalled;
```

Comment:
You may want to do a quick search through the codebase. These are required to be function calls in Jest or they won't do anything.